test/Random.test.ts

```diff
@@ -63,68 +62,47 @@ describe('RandomGenerator', function () {
     randomNumberGenerator = await RandomNumberGenerator.deploy(
       subId,
       lottery.address,
-      mockVRF.address
+      mockVRF.address,
+      lotto.chainLink.goerli.keyHash
     );
 
     await randomNumberGenerator.deployed();
 
-    let addConsumer: any = await mockVRF
+    // add consumer
+    await mockVRF
       .connect(owner)
       .addConsumer(subId, randomNumberGenerator.address);
-    addConsumer = await addConsumer.wait();
-
-    console.log(
-      'consumer added : id ',
-      subId,
-      'random address',
-      randomNumberGenerator.address
-    ); // consumer added.
 
     await lottery.initialize(randomNumberGenerator.address);
-
-    // await lottery.connect(owner).createNewLotto();
-
-    // let buy = await lottery
-    //   .connect(buyerWithAllowance)
-    //   .batchBuyLottoTicket(5, [1, 2, 3, 4, 5], nullAddress, false);
-    // let result: any = await buy.wait();
-
-    // let reqId = result.events
-    //   .filter((x: any) => x.event == "RequestNumbers")[0]
-    //   .args[1].toNumber();
-
-    // console.log(
-    //   "requestId : ",
-    //   result.events
-    //     .filter((x: any) => x.event == "RequestNumbers")[0]
-    //     .args[1].toNumber()
-    // );
-
-    // //fulfill
-    // let final = await mockVRF
-    //   .connect(owner)
-    //   .fulfillRandomWords(reqId, randomNumberGenerator.address);
-    // let final2 = await final.wait();
-    // // console.log(final2.events);
-
-    // console.log(
-    //   "random result",
-    //   (await randomNumberGenerator.getRandomResult(reqId)).toNumber()
-    // );
-
-    // console.log(res3.events);
-    // MockVRF = await ethers.getContractFactory("Mock_VRFCoordinator");
-    // mockVRF = await MockVRF.deploy();
   });
 
   describe('RequestRandomNumber', function () {
-    it('Should revert when send invalid lotteryId.', async function () {});
-    it('Should revert when send invalid roundSize input.', async function () {});
     it('Should revert when not called by Lottery address.', async function () {
-      // await chai.expect(reqId).to.equal(1);
+      await chai
```

Comment:
change import to import { expect, use as chaiUse } from 'chai';
then remove chai from all test case